app/controllers/flashcard_sets_controller.rb

```diff
@@ -0,0 +1,54 @@
+class FlashcardSetsController < ApplicationController
+  before_action :find_flashcard_set, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @flashcard_sets = policy_scope(FlashcardSet)
+  end
+
+  def new
+    @flashcard_set = FlashcardSet.new
+    authorize @flashcard_set
+  end
+
+  def create
+    @flashcard_set = FlashcardSet.new(flashcard_set_params)
+    @flashcard_set.teacher = current_teacher
+    authorize @flashcard_set
+    if @flashcard_set.save
+      redirect_to @flashcard_set
+    else
+      render :new
+    end
+  end
+
+  def show
+    authorize @flashcard_set
+  end
+
+  def edit
+    authorize @flashcard_set
+  end
+
+  def update
+    authorize @flashcard_set
+    if @flashcard_set.update(flashcard_set_params)
+      redirect_to(@flashcard_set)
```

Comment:
why the brackets? you confirmed it works? I had never seen it with brackets (and in create method you don't use them) that's why I'm asking